src/setupWorker/start/createStart.ts

```diff
@@ -15,6 +15,7 @@ import { createResponseListener } from '../../utils/worker/createResponseListene
 const DEFAULT_START_OPTIONS: DeepRequired<StartOptions> = {
   serviceWorker: {
     url: '/mockServiceWorker.js',
+    shared: false,
```

Comment:
I wouldn't put library-specific options under the `serviceWorker` key in the `StartOptions`. The intention for this key is to represent standard Service Worker options. We should probably move it to the root-level in that case.